sdks/python/src/opik/evaluation/evaluator.py

```diff
@@ -297,17 +301,23 @@ def evaluate_experiment(
 def _build_prompt_evaluation_task(
     model: base_model.OpikBaseModel, messages: List[Dict[str, Any]]
 ) -> Callable[[Dict[str, Any]], Dict[str, Any]]:
+    model_supports_vision = ModelCapabilities.supports_vision(
+        getattr(model, "model_name", None)
+    )
+
     def _prompt_evaluation_task(prompt_variables: Dict[str, Any]) -> Dict[str, Any]:
         processed_messages = []
         for message in messages:
+            rendered_content = MessageContentRenderer.render(
```

Comment:
Once we have ChatPromptTemplate class, we can immediately instantiate it from messages when evaluate_prompt is called. And here we will only need to call format and get back the processed messages list